app/gui/view/src/project.rs

```diff
@@ -281,20 +280,12 @@ impl Model {
         project_view_top_bar_size: Vector2,
     ) {
         let top_left = Vector2(-scene_shape.width, scene_shape.height) / 2.0;
-        let buttons_y = window_control_buttons::MACOS_TRAFFIC_LIGHTS_VERTICAL_CENTER;
-        let y = buttons_y - project_view_top_bar_size.y / 2.0;
-        let project_view_top_bar_origin = Vector2(0.0, y);
+        let y = -project_view_top_bar_size.y;
+        let x = ARGS.groups.window.options.top_bar_offset.value;
+        let project_view_top_bar_origin = Vector2(x as f32, y);
         self.top_bar.set_xy(top_left + project_view_top_bar_origin);
     }
 
-    fn on_close_clicked(&self) {
-        js::close(enso_config::window_app_scope_name);
```

Comment:
i'd assume this means `mod js {}` below should be removed too?

Reply:
@farmaazon I believe so, we don't need to keep this code.